spec/requests/users_spec.rb

```diff
@@ -43,6 +43,13 @@
 
         expect(response).to have_http_status(422)
       end
+
+      it 'responds with 403 status code if user does not match logged in user' do
+        create(:user)
+        get('/users/3')
+
+        expect(response).to have_http_status(403)
```

Comment:
@gabrielgaraujo I wasn't sure how to create a logged in user.
I think what I want to do here is have a logged in user with a different id try to access a different user's info. 
Presently, I am getting a 401 error instead of 403 error.

Reply:
There are two ways to do it!

You can mock the response for `JsonWebToken.decode` to return a given `user_id`
`allow(JsonWebToken).to receive(:decode) { { user_id: 3 } }`

or 

Call the `login` endpoint before you perform the `GET /users/:id`.
`post('/login', params: authentication_params)`